stylesheets/gallery.css

```diff
@@ -0,0 +1,45 @@
+h2 {
+    color: rgb(250, 218, 157);
```

Comment:
Since this color is set in styles.css, we should be able to remove that property from this ruleset.